### DISCUSSION TITLE
[Interconnect & Memory] Elastic Silicon Interconnects: Abstracting Communication in Accelerator Design

### DISCUSSION THREAD

**Discussion by teqdruid** [cucapra/latte21#29] [Papers]
Author(s): John Demme
Paper PDF: https://capra.cs.cornell.edu/latte21/paper/8.pdf
Talk Recording: https://www.youtube.com/watch?v=gjOkGX2E7EY

**Comment by sahandKashani** [comment]
Hey John. Just wanted to say that I love the narration in your video :)
